### DISCUSSION TITLE
regarding dataset

### DISCUSSION THREAD

**Discussion by Aditya04-code** [anujdutt9/Disease-Prediction-from-Symptoms#14] [General]
Hi anuj can you please provide us with the datasets as i am working on a project based on that it would be preety great help of yours :)

**Comment by anujdutt9** [comment]
Hi @Aditya04-code , the dataset is available here: https://github.com/anujdutt9/Disease-Prediction-from-Symptoms/tree/master/dataset
I scraped the dataset off from this source: https://impact.dbmi.columbia.edu/~friedma/Projects/DiseaseSymptomKB/index.html
Let me know if you still have issues.

**Comment by Aditya04-code** [comment]
Thank you so much anuj

**Comment by Aditya04-code** [comment]
Hey Anuj I just made the project based on a Healthcare - Support System
https://aditya04-code-minor-p-app-4uhjh1.streamlit.app/
please rate our project



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
05/17/23,
12:34:29 AM
